pkg/fqdn/cell/cell.go

```diff
@@ -22,5 +24,11 @@ var Cell = cell.Module(
 	// The FQDN bootstrap logic
 	bootstrap.Cell,
 
+	// The FQDN Message handler
```

Comment:
We could maybe phrase this better, e.g. to mention that this is used to process the DNS message etc. Just stating the name of it doesn't really help the reader to understand what it does.